packages/cdktf-cli/bin/cmds/helper/init.ts

```diff
@@ -145,11 +145,17 @@ This means that your Terraform state file will be stored locally on disk in a fi
           "utf8"
         );
 
+        const { terraformModules, terraformProviders } = JSON.stringify(
+          cdktfJson
+        ) as any;
```

Comment:
Why not just check the object directly?

Reply:
you mean `config.terraformModules` etc?

Reply:
I mean why convert `cdktfJson` to a string when it is already an object?